### DISCUSSION TITLE
[HW8] What instruction should be covered?

### DISCUSSION THREAD

**Discussion by kohs100** [prosyslab-classroom/cs524-program-analysis#440] [Q&A] [answered]
getelementptr, sext, ptrtoint 등 지금까지의 과제에서 다루지 않았던 형변환/포인터 관련 명령어, 테스트 코드 컴파일시 생성되지 않는 것으로 보이는 phi 인스트럭션 등의 테스트 코드 포함 여부가 공개 가능한지 궁금합니다.

I wonder if it is possible to disclose whether test code contains special instructions such as getelementptr, sext, ptrtoint, etc., which were not covered in previous tasks, and phi instruction that does not appear to be generated when compiling test code.

**Comment by goodtaeeun** [comment, marked as answer]
안녕하세요,
- phi 인스트럭션은 등장하지 않습니다.
- sext, zext등 기존에 다루지 않았던 인스트럭션등은 테스트 케이스에서 등장할 수 있습니다. 그러나 이 경우 nop으로 간주하고 해당 인스트럭션으로부터는 아무 정보도 extract 안 하셔도 됩니다.

---

Hello,
- phi instructions will not appear in the test cases.
- Some instructions not covered by previous assignments may appear in the test cases (i.e., sext, zext). However, you may regard them as nop and not extract any information from such instructions.
